ParabolInc/parabol
# feat(orgAdmins): Make org members view sortable

# Description

Sort organization members by email (alphabetically) or last seen date. Discussion of the design iteration can be found [here](https://parabol.slack.com/archives/C029ZTH0EL9/p1724193938697549)

## Demo

![2024-08-22 11 36 23](https://github.com/user-attachments/assets/e22462e0-cfb8-4372-b540-347dec6fb738)


## Testing scenarios

- [ ] Sortable org members view
  - Go to the org members page: `http://localhost:3000/me/organizations/xxxx/members`
  - Verify that you can sort the members by their emails
  - Verify that you can sort the members by last seen date

## Final checklist
